tests/end_to_end/test_artifacts.py

```diff
@@ -86,7 +82,7 @@ def test_catalog_shows_all_versions(execute):
 from time import sleep
 x = 1
 lineapy.save(x, "x")
-sleep(1)
+x = 1
```

Comment:
maybe use a diff value of for 2nd version here and add an assert to verify that the value is getting updated correctly (i think there was change in logic for node saving above)